qi-lib/flow/core/compiler/deforest/cps.rkt

```diff
@@ -21,7 +21,8 @@
 (define-syntax inline-compose1
```

Comment:
RE: 6bafcfc05c00957f20256df783da1d4596c8da9a's message, perhaps "add source syntax context to transformer composition"?